include/boost/geometry/strategies/distance/comparable.hpp

```diff
@@ -0,0 +1,61 @@
+// Boost.Geometry
+
+// Copyright (c) 2021, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Adam Wulkiewicz, on behalf of Oracle
+
+// Licensed under the Boost Software License version 1.0.
+// http://www.boost.org/users/license.html
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_DISTANCE_COMPARABLE_HPP
+#define BOOST_GEOMETRY_STRATEGIES_DISTANCE_COMPARABLE_HPP
+
+
+#include <boost/geometry/core/cs.hpp>
+#include <boost/geometry/core/static_assert.hpp>
+
+
+namespace boost { namespace geometry
+{
+
+namespace strategies { namespace distance
+{  
+
+#ifndef DOXYGEN_NO_DETAIL
+namespace detail
+{
+
+template <typename Strategies>
+struct comparable
+    : public Strategies
```

Comment:
What do you mean? Aren't we indent base classes list like that? Is it because in this case there really no need for that so the base class could be at the same line as the derived class? Or do you mean something else?

Reply:
I was thinking that there is no need in this case, as for example here 
https://github.com/boostorg/geometry/blob/develop/include/boost/geometry/strategies/area/cartesian.hpp#L27
but now I am thinking it again I will agree with you to be consistent and always change line.

Reply:
Ok, I plan to alter the strategies in other PRs so I'll make the cartesian area strategy consistent.